README.adoc

```diff
@@ -2,188 +2,184 @@
+handle these requirements, as the following listing shows:
 
 ====
 [source,properties]
 ----
-include::complete/eureka-service/src/main/resources/application.properties[]
+include::eureka-server/src/main/resources/application.properties[]
 ----
 ====
 
+You can now run the Eureka Server with the command `./gradlew bootRun`.
+
 == Talking to the Registry
 
-Now that you have started a service registry, you can stand up a client that both
-registers itself with the registry and uses the Spring Cloud `DiscoveryClient` abstraction
-to interrogate the registry for its own host and port. The `@EnableDiscoveryClient`
-activates the Netflix Eureka `DiscoveryClient` implementation. (There are other
-implementations for other service registries, such as
-https://www.consul.io/[Hashicorp's Consul] or https://zookeeper.apache.org/[Apache Zookeeper]).
-The following listing (from
-`eureka-client/src/main/java/example/serviceregistrationanddiscoveryclient/ServiceRegistrationAndDiscoveryClientApplication.java`)
-shows the client application:
+Now that you have started a service registry, you can stand up clients that interact with the registry. Our client applications, ServiceA and ServiceB, will automatically register with the Eureka server because we have `spring-cloud-starter-netflix-eureka-client` on the classpath.
+To avoid port conflicts, set the `server.port` parameter in both ServiceA and ServiceB:
 
+Service A:
 ====
-[source,java]
+[source,properties]
 ----
-include::complete/eureka-client/src/main/java/com/example/serviceregistrationanddiscoveryclient/ServiceRegistrationAndDiscoveryClientApplication.java[]
+include::servicea/src/main/resources/application.properties[]
 ----
 ====
 
-Whatever implementation you choose, you should soon see `eureka-client` registered under
-whatever name you specify in the `spring.application.name` property. This property is used
-a lot in Spring Cloud, often in the earliest phases of a service's configuration. This
-property is used in service bootstrap and, so, by convention lives in
-`eureka-client/src/main/resources/bootstrap.properties` where it is found before
-`src/main/resources/application.properties`. The following listing shows the
-`bootstrap.properties` file:
-
+Service B:
 ====
 [source,properties]
 ----
-include::complete/eureka-client/src/main/resources/bootstrap.properties[]
+include::serviceb/src/main/resources/application.properties[]
 ----
 ====
 
-The `eureka-client` defines a Spring MVC REST endpoint (`ServiceInstanceRestController`)
-that returns an enumeration of all the registered `ServiceInstance` instances at
-`http://localhost:8080/service-instances/a-bootiful-client`. See the
-https://spring.io/guides/gs/rest-service/[Building a RESTful Web Service]  guide to learn
-more about building REST services with Spring MVC and Spring Boot.
+At this point you should be able to run all three applications.
+You can use the IDE or execute the command `./gradlew bootRun` from each application folder.
```

Comment:
Move command to after the command.

So replace "command `./gradlew bootRun`" with "`./gradlew bootRun` command"

The other way to do it is "command, `./gradlew bootRun`," (that is, wrapping the command in commas). This has to do with English's ordering of nouns. If you shift from the usual order, you have to make it be parenthetical, so you need commas. I run into this constantly, because people speak in this order (ignoring the commas when speaking), but it's incorrect to write in this order.